src/lib/storage/frame_of_reference_segment/frame_of_reference_segment_iterable.hpp

```diff
@@ -101,6 +101,7 @@ class FrameOfReferenceSegmentIterable : public PointAccessibleSegmentIterable<Fr
 
     void advance(std::ptrdiff_t n) {
       // For now, the lazy approach
+      PerformanceWarning("using repeated increment/decrement for random access");
```

Comment:
Wouldn't make you re-run CI for that, but since we have to do that anyway: Capitalize 'Using'.